tools/plugins/com.liferay.ide.project.core/templates/basetemplate.ftl

```diff
@@ -0,0 +1,5 @@
+package ${packagename};
```

Comment:
Don't add these to the /templates/ folder at the root of the bundle but instead add them to the "modules.template" package and load them from there.